### ISSUE TITLE
REGEXP_REPLACE trim last character

### ISSUE THREAD

**Issue by vespadj** [nalgeon/sqlean#45]
```sql
SELECT load_extension('./re.so'); 
-- .load ./re
SELECT REGEXP_REPLACE('this year is 2021', '[0-9]+', '2050');
```

This returns `205` instead of `2050`.

Linux. SQLite version 3.22.0

**Comment by nalgeon** [comment]
Thanks for reporting! There is definitely a problem with older versions of SQLite. Will investigate.

**Comment by vespadj** [comment]
For my testing:
1) on CLI, I install by `sudo apt-get install sqlite3` , (v.3.22.0)
2) GUI https://sqlitebrowser.org/  - sqlite v.3.35.5
On sqlite.org, Latest Release Version 3.39.2 (2022-07-21).

Can you suggest another program with GUI than supports `SELECT load_extension('./re.so')` ?

**Comment by nalgeon** [comment]
Fixed in [0.15.3](https://github.com/nalgeon/sqlean/releases/tag/0.15.3).
